### DISCUSSION TITLE
Problemas VSCode

### DISCUSSION THREAD

**Discussion by gbrlbrt** [clube-de-leitura-python/pense-em-python-2021#17] [Perguntas & Respostas] [answered]
Olá, estou com um problema com meu VSCode. Quando eu rodo um arquivo usando shift+Enter não tenho problemas, mas nunca consigo rodar pelo botão verde. Sempre me dá essa mensagem:
>>> /usr/bin/python3 /Users/gbb/Desktop/PyTut/grade.py
  File "<stdin>", line 1
    /usr/bin/python3 /Users/gbb/Desktop/PyTut/grade.py
    ^
SyntaxError: invalid syntax
>>> 

PS: estou usando um mac

Alguém consegue me ajudar?

**Comment by vitorestevam** [comment]
Encontrei essa pergunta aqui no stack overflow que parece ser bem o teu problema. Tenta dar uma olhada 😉
https://stackoverflow.com/questions/51540391/invalid-syntax-error-when-running-python-from-inside-visual-studio-code

**Reply by gbrlbrt** [reply]
Opa, valeu. Achei o problema, tinha que desativar o terminal.integrated.inheritEnv , mas não faço ideia do que isso faça.
Abs!

**Comment by gbrlbrt** [comment, marked as answer]
Update: estava rodando o código, mas ainda aparecia mensagem de erro. Como tava irritando, fui atrás de uma solução. Instalei o pylint pelo terminal e resolveu! Obrigado pela ajuda.
